blog/zh/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md

```diff
@@ -1,50 +1,56 @@
 ---
-title: "GCP、AWS 和 Azure ARM 架构服务器性能测试对比"
+title: "GCP、AWS、Azure 和 Oracle ARM 架构服务器性能测试对比"
 authors:
   - name: "赵士瑞"
     title: "Author"
-    url: "https://github.com/soulbird"
+    url: "soulbird - Overview"
     image_url: "https://github.com/soulbird.png"
   - name: "苏钰"
     title: "Technical Writer"
-    url: "https://github.com/SylviaBABY"
+    url: "SylviaBABY - Overview"
```

Comment:
wrong `url`s, please revert back to something like

`https://github.com/soulbird` and `https://github.com/SylviaBABY`